dist/_include-media.scss

```diff
@@ -139,6 +222,32 @@ $unit-intervals: (
 ///
 @function noop() {}
 
+@function im-intercepts-static-breakpoint($conditions...) {
```

Comment:
No, but I think it would be a good principle moving forwards. Are you okay with it, or do you prefer not having the two approaches coexisting — i.e. leave this one unprefixed or prefix them all?